jekyll/_includes/snippets/env-var-or-context.adoc

```diff
@@ -0,0 +1 @@
+It is best practice to set environment variables at the <<env-vars#setting-an-environment-variable-in-a-project,project level>> or within a <<env-vars#setting-an-environment-variable-in-a-context,context>>.
```

Comment:
There's a hidden concept here which is that it's a best practice because these are secret keys. Is there a way to say that explicitly in this snippet?

Reply:
I will add a link to https://circleci.com/docs/2.0/contexts/#secrets-masking